src/h5p-standalone.ts

```diff
@@ -201,12 +202,23 @@ export class H5PStandalone {
         /**
          * Load H5P content and libraries
          */
-        const { contentAsJson } = options
+        const { contentAsJson, h5pAsJson } = options;
         const {h5pJsonPath, contentJsonPath, librariesPath} = this.getH5PPaths(options);
 
-        const H5PJsonContent = <H5PPackageDefinition>(await getJSON(`${h5pJsonPath}/h5p.json`, options?.assetsRequestFetchOptions).catch((e) => {
-            console.log('Error while trying to fetch h5p json content: ', e, `${h5pJsonPath}/h5p.json`, options?.assetsRequestFetchOptions)
-        }));
+        let H5PJsonContent;
+        if (h5pAsJson) {
+            try {
+                H5PJsonContent = JSON.parse(h5pAsJson);
+            } catch (e) {
+                H5PJsonContent = <H5PPackageDefinition>(await getJSON(`${h5pJsonPath}/h5p.json`, options?.assetsRequestFetchOptions).catch((e) => {
+                    console.log('Error while trying to fetch h5p json content: ', e, `${h5pJsonPath}/h5p.json`, options?.assetsRequestFetchOptions)
```

Comment:
Please remove this fetch fallback.

If the `h5pAsJson` is provided, it **must** be valid.

On catch, instead of making the fetch, please throw an exception describing that an invalid json content was provided in the `h5pAsJson` property.

This will make the system more stable and understandable.